main/resource-access.bs

```diff
@@ -142,19 +139,151 @@ through POST, using the HTTP `Slug` header as defined in [[!RFC5023]].
 Clients who want the server to determine the URI for their resource, MUST use
 the POST request.
 
-Servers MUST allow creating new resources with a POST request to URI path
+[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)].
+
+### Reading resources ### {#read}
+
+Servers MUST support the HTTP `GET`, `HEAD` and `OPTIONS` methods [[!RFC7231]]
+for clients to read resources or to determine communication options.
+[[Source](https://github.com/solid/specification/issues/39#issuecomment-538017667)]
+
+When using Web Access Control ([[#wac]]):
+
+Servers MUST indicate their support for HTTP Methods by responding to HTTP
+`GET`, `HEAD` and `OPTIONS` requests on the resource's URL with the HTTP
+Method tokens in the HTTP response header `Allow`.
+
+Servers MUST indicate supported media types in the `Accept-Patch` [[!RFC5789]]
+and the `Accept-Post` [[!LDP]] headers for matching HTTP methods in `Allow`
+header value.
+
+Servers MAY include the HTTP `Accept-Patch` and `Accept-Post` headers in the
+response of a `OPTIONS *` request.
+
+[[Source](https://github.com/solid/specification/issues/85#issuecomment-575386251)]
+[[Source](https://github.com/solid/specification/issues/43)]
+
+### Writing resources ### {#write}
+
+Servers MUST support the HTTP `PUT`, `POST` and `PATCH` methods [[!RFC7231]]
+for clients to create and update resources.
```

Comment:
What if my server is read-only?
I don't think it is a MUST, but rather a MAY.

Reply:
See https://github.com/solid/specification/issues/39#issuecomment-533298070

Reply:
Unless of course `405` is an an acceptable implementation.

Reply:
https://github.com/solid/specification/pull/188/commits/4d5cb9bac37e8206d267bf3df39a21840383b657

Reply:
https://github.com/solid/specification/pull/188/commits/09cc77a20cab781177e18e4b970cd4e3426ee5d0 should ties this up.